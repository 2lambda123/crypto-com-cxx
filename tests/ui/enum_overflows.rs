#[cxx::bridge]
mod ffi {
    enum Good1 {
        A = 0xffffffff,
    }
    enum Good2 {
        B = 0xffffffff,
        C = 2020,
    }
    enum Bad {
        D = 0xfffffffe,
        E,
        F,
    }
}

fn main() {}
