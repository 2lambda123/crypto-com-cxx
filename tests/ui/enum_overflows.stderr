error: overflowed on value
  --> $DIR/enum_overflows.rs:13:9
   |
13 |         F,
   |         ^
